### PR TITLE
SR2024 kick off email

### PR DESCRIPTION
**Pull request by sedders123**
Still needs spellchecking, de-memeifying, and some links adding but this is what came out of the doing this evening

### CODE REVIEW

File: SR2024/2023-07-15-sr2024-kickofff.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- 2024-03-23 – 2024-03-24 Competition (provisional)
+
+## Get involved
+
+There's loads of ways to help with SR2024, taking as much or as little time as you can give.
+
+- The physical competition needs a small army of volunteers to run in numerous techincal and non technical roles. We'll open sign ups for volunteering at the event as soon as we can confirm the venue ???
+- We're on the look out for mentors, who would regularly attend a teams' meetings (either virtually on in person) and assist them
+- We'll be running Tech Days and in person events which will need people to help run
+- We'll soon be packing the kits for shipping which will require a lot of hands (no technical knowledge required!)
```
Comment (Adimote) [inline SR2024/2023-07-15-sr2024-kickofff.md:24]:
```suggestion
```
Delete this because it's now in the past

**Review by Adimote — APPROVED** [review]

**Review by Adimote — CHANGES_REQUESTED** [review]
unapproving because I only approved accidentally

File: SR2024/2023-07-15-sr2024-kickofff.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+### Teams
+
+Running Student Robotics is no small feat so to help spread the load we've organised into teams, each with their own area of interest.
+
+Committee| Description | Email | Current Meeting Time (UK time)
+---|---|---|---|
+Competition| Organise the competiton | competition-committee@studentrobotics.org| Wednesday 7:30pm
+Kit| They do the kit | kit-committee@studentrobotics.org | Tuesday 7pm
+Marketing| Social Media + crocs | marketing-committee@studentrobotics.org| Thursday 7:30pm
+Infrastructure| Hit servers | infrastructure-committee@studentrobotics.org| Wednesday 7pm (monthly)
```
Comment (PeterJCLaw) [inline SR2024/2023-07-15-sr2024-kickofff.md:36]:
```suggestion
Infrastructure| Hit servers | infrastructure-committee@studentrobotics.org| Monday 7:30pm (monthly)
```

File: SR2024/2023-07-15-sr2024-kickofff.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+Volunteer Onboarding| Welcome wagon | onboarding@studentrobotics.org | TBC
```
Comment (PeterJCLaw) [inline SR2024/2023-07-15-sr2024-kickofff.md:37]:
```suggestion
Volunteer Onboarding| Welcome wagon | onboarding@studentrobotics.org | Tuesday 8pm (fortnightly)
```

File: SR2024/2023-07-15-sr2024-kickofff.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+Fundraising| Moollah | trustees@studentrobotics.org| Mondays 8:30pm
```
Comment (PeterJCLaw) [inline SR2024/2023-07-15-sr2024-kickofff.md:38]:
```suggestion
Fundraising| Moollah | trustees@studentrobotics.org| Monday 8:30pm

```

File: SR2024/2023-07-15-sr2024-kickofff.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+If you want to help out throughout the year, email the relevant committee or ask in Slack. No prior expereience is necessary, just a can do attitude!
+
+
+## Stay up to date
+
+If you want to keep up to date with the latest happenings in Student Robotics then you can join our Slack. Most planning and ad-hoc communication occurs there and you'll be able to get stuck in if you want to!
```
Comment (Adimote) [inline SR2024/2023-07-15-sr2024-kickofff.md:45]:
```suggestion
If you want to keep up to date with the latest happenings in Student Robotics then you can [join our Slack](https://forms.gle/DsM45qByLA3heZELA). Most planning and ad-hoc communication occurs there and you'll be able to get stuck in if you want to!

```

File: SR2024/2023-07-15-sr2024-kickofff.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+You could also subscribe to SR(A)WN, our (almost) Weekly Newsletter. This contains headline updates from each Team and can be subscribed to via RSS or Google Groups.
```
Comment (PeterJCLaw) [inline SR2024/2023-07-15-sr2024-kickofff.md:47]:
```suggestion
You could also subscribe to [SR(A)WN](https://studentrobotics.org/srawn/), our (almost) Weekly Newsletter. This contains headline updates from each Team and can be subscribed to via [RSS](https://studentrobotics.org/srawn/rss.xml) or [Google Groups](https://groups.google.com/g/srawn).

```

File: SR2024/2023-07-15-sr2024-kickofff.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+- 2023-10-21 Kickstart
+- 2024-02-10 Virtual competition (provisional)
+- 2024-03-23 – 2024-03-24 Competition (provisional)
+
+## Get involved
+
+There's loads of ways to help with SR2024, taking as much or as little time as you can give.
+
+- The physical competition needs a small army of volunteers to run in numerous technical and non technical roles. We'll open sign ups for volunteering at the event as soon as we can confirm the venue ???
```
Comment (PeterJCLaw) [inline SR2024/2023-07-15-sr2024-kickofff.md:21]:
```suggestion
- The physical competition needs a small army of volunteers to run in numerous technical and non technical roles. We'll open sign ups for volunteering at the event as soon as we can confirm the venue later in the year

```

File: SR2024/2023-07-15-sr2024-kickofff.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
+
+Hi All,
+
+
+Planning for SR2024 is underway! We're still in the early stages so this is the perfect time for you to get more involved.
```
Comment (PeterJCLaw) [inline SR2024/2023-07-15-sr2024-kickofff.md:5]:
```suggestion
Planning for SR2024 is underway! With Kickstart just around the corner now is a great time to get involved.
```

**Comment by antoinepetty** [comment]
Looks good to me, but Onboarding meet weekly :) (or at least try to)

**Comment by PeterJCLaw** [comment]
> Looks good to me, but Onboarding meet weekly :) (or at least try to)

a0a4e70

**Comment by antoinepetty** [comment]
Also fundraising have fundraising@studentrobotics.org
I'd be tempted to put them above onboarding as they're more desperate for volunteers

**Comment by PeterJCLaw** [comment]
> Also fundraising have [fundraising@studentrobotics.org](mailto:fundraising@studentrobotics.org)

I suspect the goal here was to include the email address of the relevant committees rather than the teams.

**Comment by antoinepetty** [comment]
> I suspect the goal here was to include the email address of the relevant committees rather than the teams.

Ah we might want volunteer-onboarding-committee@studentrobotics.org for onboarding then? I don't mind onboarding@ though, much easier to type!

**Comment by PeterJCLaw** [comment]
> > I suspect the goal here was to include the email address of the relevant committees rather than the teams.
> 
> Ah we might want [volunteer-onboarding-committee@studentrobotics.org](mailto:volunteer-onboarding-committee@studentrobotics.org) for onboarding then? I don't mind onboarding@ though, much easier to type!

I had a look at that, however volunteer-onboarding-committee@ doesn't seem to allow posts from external people?

**Comment by antoinepetty** [comment]
> I had a look at that, however volunteer-onboarding-committee@ doesn't seem to allow posts from external people?

Ah I don't think that's intentional, I've just modified the team address to allow external emails but don't have access to modify the committee settings. Happy to keep it as onboarding@ here though

**Review by Adimote — APPROVED** [review]

**Pull request merged**